kai_mcp_solution_server/src/kai_mcp_solution_server/server.py

```
@@ -458,39 +648,177 @@
+            solution.update_solution_status()
+            session.add(solution)
+            await session.flush()
+
+            # print(
+            #     f"Solution {solution.id} status: {solution.solution_status}",
+            #     file=sys.stderr,
+            # )
+
+            if not (
+                solution.solution_status == SolutionStatus.ACCEPTED
+                or solution.solution_status == SolutionStatus.MODIFIED
```

Pylint (3.3.7)

[refactor] Consider merging these comparisons with 'in' by using 'solution.solution_status in (SolutionStatus.ACCEPTED, SolutionStatus.MODIFIED)'. Use a set instead if elements are hashable.

(R1714)